### DISCUSSION TITLE
Image thumbnails still being created in Windows with "video only" preset...

### DISCUSSION THREAD

**Discussion by avekifes** [Xanashi/Icaros#38] [Q&A]
Is this normal behavior? File Explorer Options is set to "always show icons, never thumbnails."

**Comment by avekifes** [comment]
Never mind; I was able to fix this by unchecking and rechecking the relevant boxes in File Explorer Options.

**Comment by avekifes** [comment]
...and the problem is back after a restart.

**Comment by Xanashi** [comment]
Hi avekifes,

You're mentioning two different options here, so I'll do a two part answer:

1. The 'Video Only' preset in IcarosConfig, makes it so that Icaros will only provide thumbnails for video files. 
The preset doesn't disable thumbnails for all other filetypes though, as other shell extension are still working normally.
It only affect the files Icaros will be handling in File Explorer.

2. The File Explorer option; "always show icons, never thumbnails.", is a native Windows option, that shell extensions (like Icaros),
have no impact on at all. Once this is enabled, no thumbnails should be shown at all in File Explorer, only icons. 
Icaros Thumbnailing is basically disabled, when this option is enabled. 

Please note though, that when you enable Icaros Thumbnailing in IcarosConfig, Icaros will automatically disable this setting, 
because it assumes the user wants thumbnails to be displayed in File Explorer.
